### ISSUE TITLE
unstable tag -  No module named 'pkg_resources

### ISSUE THREAD

**Issue by stevezau** [linuxserver/docker-sabnzbd#106]
When running the latest unstable tag i am getting the below error. If i revert back to latest tag it works.

```
ModuleNotFoundError: No module named 'pkg_resources'
Traceback (most recent call last):
  File "/app/sabnzbd/SABnzbd.py", line 56, in <module>
    import sabnzbd
  File "/app/sabnzbd/sabnzbd/__init__.py", line 91, in <module>
    import sabnzbd.rss as rss
  File "/app/sabnzbd/sabnzbd/rss.py", line 35, in <module>
    import sabnzbd.emailer as emailer
  File "/app/sabnzbd/sabnzbd/emailer.py", line 36, in <module>
    from sabnzbd.notifier import check_cat
  File "/app/sabnzbd/sabnzbd/notifier.py", line 36, in <module>
    from sabnzbd.newsunpack import create_env
  File "/app/sabnzbd/sabnzbd/newsunpack.py", line 61, in <module>
    from sabnzbd.sorting import SeriesSorter
  File "/app/sabnzbd/sabnzbd/sorting.py", line 25, in <module>
    import guessit
  File "/usr/local/lib/python3.6/dist-packages/guessit/__init__.py", line 8, in <module>
    from .api import guessit, GuessItApi
  File "/usr/local/lib/python3.6/dist-packages/guessit/api.py", line 17, in <module>
    from .rules import rebulk_builder
  File "/usr/local/lib/python3.6/dist-packages/guessit/rules/__init__.py", line 11, in <module>
    from .properties.episodes import episodes
  File "/usr/local/lib/python3.6/dist-packages/guessit/rules/properties/episodes.py", line 16, in <module>
    from .title import TitleFromPosition
  File "/usr/local/lib/python3.6/dist-packages/guessit/rules/properties/title.py", line 11, in <module>
    from .language import (
  File "/usr/local/lib/python3.6/dist-packages/guessit/rules/properties/language.py", line 10, in <module>
    import babelfish
  File "/usr/local/lib/python3.6/dist-packages/babelfish/__init__.py", line 20, in <module>
    from .converters import (LanguageConverter, LanguageReverseConverter, LanguageEquivalenceConverter, CountryConverter,
  File "/usr/local/lib/python3.6/dist-packages/babelfish/converters/__init__.py", line 6, in <module>
    from pkg_resources import iter_entry_points, EntryPoint
ModuleNotFoundError: No module named 'pkg_resources'
Traceback (most recent call last):
  File "/app/sabnzbd/SABnzbd.py", line 56, in <module>
    import sabnzbd
  File "/app/sabnzbd/sabnzbd/__init__.py", line 91, in <module>
    import sabnzbd.rss as rss
  File "/app/sabnzbd/sabnzbd/rss.py", line 35, in <module>
    import sabnzbd.emailer as emailer
  File "/app/sabnzbd/sabnzbd/emailer.py", line 36, in <module>
    from sabnzbd.notifier import check_cat
  File "/app/sabnzbd/sabnzbd/notifier.py", line 36, in <module>
    from sabnzbd.newsunpack import create_env
  File "/app/sabnzbd/sabnzbd/newsunpack.py", line 61, in <module>
    from sabnzbd.sorting import SeriesSorter
  File "/app/sabnzbd/sabnzbd/sorting.py", line 25, in <module>
    import guessit
  File "/usr/local/lib/python3.6/dist-packages/guessit/__init__.py", line 8, in <module>
    from .api import guessit, GuessItApi
  File "/usr/local/lib/python3.6/dist-packages/guessit/api.py", line 17, in <module>
    from .rules import rebulk_builder
  File "/usr/local/lib/python3.6/dist-packages/guessit/rules/__init__.py", line 11, in <module>
    from .properties.episodes import episodes
  File "/usr/local/lib/python3.6/dist-packages/guessit/rules/properties/episodes.py", line 16, in <module>
    from .title import TitleFromPosition
  File "/usr/local/lib/python3.6/dist-packages/guessit/rules/properties/title.py", line 11, in <module>
    from .language import (
  File "/usr/local/lib/python3.6/dist-packages/guessit/rules/properties/language.py", line 10, in <module>
    import babelfish
  File "/usr/local/lib/python3.6/dist-packages/babelfish/__init__.py", line 20, in <module>
    from .converters import (LanguageConverter, LanguageReverseConverter, LanguageEquivalenceConverter, CountryConverter,
  File "/usr/local/lib/python3.6/dist-packages/babelfish/converters/__init__.py", line 6, in <module>
    from pkg_resources import iter_entry_points, EntryPoint
ModuleNotFoundError: No module named 'pkg_resources'
Traceback (most recent call last):
  File "/app/sabnzbd/SABnzbd.py", line 56, in <module>
    import sabnzbd
  File "/app/sabnzbd/sabnzbd/__init__.py", line 91, in <module>
    import sabnzbd.rss as rss
  File "/app/sabnzbd/sabnzbd/rss.py", line 35, in <module>
    import sabnzbd.emailer as emailer
  File "/app/sabnzbd/sabnzbd/emailer.py", line 36, in <module>
    from sabnzbd.notifier import check_cat
  File "/app/sabnzbd/sabnzbd/notifier.py", line 36, in <module>
    from sabnzbd.newsunpack import create_env
  File "/app/sabnzbd/sabnzbd/newsunpack.py", line 61, in <module>
    from sabnzbd.sorting import SeriesSorter
  File "/app/sabnzbd/sabnzbd/sorting.py", line 25, in <module>
    import guessit
  File "/usr/local/lib/python3.6/dist-packages/guessit/__init__.py", line 8, in <module>
    from .api import guessit, GuessItApi
  File "/usr/local/lib/python3.6/dist-packages/guessit/api.py", line 17, in <module>
    from .rules import rebulk_builder
  File "/usr/local/lib/python3.6/dist-packages/guessit/rules/__init__.py", line 11, in <module>
    from .properties.episodes import episodes
  File "/usr/local/lib/python3.6/dist-packages/guessit/rules/properties/episodes.py", line 16, in <module>
    from .title import TitleFromPosition
  File "/usr/local/lib/python3.6/dist-packages/guessit/rules/properties/title.py", line 11, in <module>
    from .language import (
  File "/usr/local/lib/python3.6/dist-packages/guessit/rules/properties/language.py", line 10, in <module>
    import babelfish
  File "/usr/local/lib/python3.6/dist-packages/babelfish/__init__.py", line 20, in <module>
    from .converters import (LanguageConverter, LanguageReverseConverter, LanguageEquivalenceConverter, CountryConverter,
  File "/usr/local/lib/python3.6/dist-packages/babelfish/converters/__init__.py", line 6, in <module>
    from pkg_resources import iter_entry_points, EntryPoint
ModuleNotFoundError: No module named 'pkg_resources;
```

**Comment by aptalca** [comment]
looks like the latest beta/unstable build is broken and likely an upstream issue

**Comment by aptalca** [comment]
Fixed in #107

**Comment by stevezau** [comment]
confirmed working, thanks
